benchmark/cluster/vgg16/README.md

```diff
@@ -0,0 +1,58 @@
+# Performance for distributed vgg16
+
+## Test Result
+
+### Single node single thread
+
+| Batch Size | 32 | 64 | 128 | 256 |
+| -- | -- | -- | -- | -- |
+| PaddlePaddle Fluid | - | - | 16.74 | - |
+| PaddlePaddle v2 | - | - | 17.60 | - |
+| TensorFlow | - | - | - | - |
+
+### different batch size
+
+- PServer Count: 10
+- Trainer Count: 20
+- Metrics: samples / sec
+
+| Batch Size | 32 | 64 | 128 | 256 |
+| -- | -- | -- | -- | -- |
+| PaddlePaddle Fluid | - | 247.40 | - | - |
```

Comment:
It seesm fluid's performance is `247.40/64=3.866` batch per second, and v2's performance is `256.14/128=2.001` batch per second.
Seems the different is huge, do you have an idea why? (also please check if my math is correct).

Reply:
Sorry, wrong column. I'll update this PR with full test result.